image_publisher:
  ros__parameters:
    image_topic: "/image"
    map_frame_id: "map"

image_to_gridmap:
  ros__parameters:
    resolution: 0.03
    min_height: -0.1
    max_height: 0.1

grid_map_visualization:
  ros__parameters:
    grid_map_topic: /grid_map
    grid_map_visualizations: [elevation_points, elevation_grid, flat_grid]
    elevation_points:
      type: point_cloud
      params:
        layer: elevation
    elevation_grid:
      type: occupancy_grid
      params:
        layer: elevation
        data_min: -0.2
        data_max: 0.2
    flat_grid:
      type: flat_point_cloud
      params:
        height: 0.0
